compose/ui/ui/src/desktopMain/kotlin/androidx/compose/ui/scene/ComposeContainer.desktop.kt

```diff
@@ -155,6 +155,7 @@ internal class ComposeContainer(
     }
 
     fun dispose() {
+        _windowContainer?.removeComponentListener(this)
```

Comment:
Let's reuse setter to have unsubscription in one place
```suggestion
        windowContainer = null
```

Reply:
It doesn't work, because the property isn't nullable:
```
var windowContainer: JLayeredPane
```